docs/fx-core/bot-help.md

```diff
@@ -52,22 +52,3 @@ Suppose an existing AAD has been registered, you can reuse this AAD by configuri
     }
 ```
 4. Provision the Bot, then it will reuse this AAD and won't create a new one.
-
-## Configuration schema for bot
```

Comment:
So all these config values are no applicable any more?

Reply:
No, it's removed since it's duplicated with config.md.

Reply:
The updated section resides in config.md now.